core/iwasm/aot/aot_loader.c

```diff
@@ -3269,3 +3324,25 @@ aot_get_plt_table_size()
 {
     return get_plt_table_size();
 }
+
+#if WASM_ENABLE_LOAD_CUSTOM_SECTION != 0
+const uint8 *
+aot_get_custom_section(AOTModule *module, const char *name, uint32 *len)
+{
+    WASMCustomSection *section = module->custom_section_list;
+
+    while (section) {
+        if ((section->name_len == strlen(name))
+            && (memcmp(section->name_addr, name, section->name_len) == 0)) {
```

Comment:
Can just use `strcmp(section->section_name, name)`, but current code also works.

Reply:
Yes, things are much more easier when the string is NULL terminated in AoT, I've changed to strcmp